Source/WebCore/css/CSSSelector.cpp

```diff
@@ -866,4 +866,28 @@ bool CSSSelector::hasExplicitPseudoClassScope() const
     return visitAllSimpleSelectors(check);
 }
 
+bool CSSSelector::isHostPseudoClass() const
+{
+    return match() == Match::PseudoClass && pseudoClass() == PseudoClass::Host;
+}
+
+bool CSSSelector::isOrSimilarToHostPseudoClass() const
```

Comment:
Would `matchesHostPseudoClass()`  or `containsMatchForHostPseudoClass()` be more descriptive?

Reply:
Function removed